packages/core-modules/contracts/modules/mocks/SampleFeatureFlagModule.sol

```diff
@@ -0,0 +1,17 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../interfaces/ISampleFeatureFlagModule.sol";
+import "../../storage/SampleStorage.sol";
+import "../../storage/FeatureFlag.sol";
+
+contract SampleFeatureFlagModule is ISampleFeatureFlagModule {
```

Comment:
fyi sample/mock modules like this will be replaced with some sort of codegen in the near future so don't invest a ton of time in this.

Reply:
sweet 👍🏽 ; pretty basic here